### PR TITLE
Get proposal response payload from chaincode action

### PR DESCRIPTION
**Pull request by bestbeforetoday**
This is the correct chaincode transaction response payload location. The response payload field is intended to contain metadata but the peer currently copies the transaction response payload here for convenience. This causes failures due to gRPC message size limits for large response payloads and so may be removed in the future.

Also changed a few build scripts to honour the GO_CMD environment variable.

### CODE REVIEW

File: pkg/client/channel/invoke/txnhandler.go [hunk 1]
```diff
@@ -63,7 +64,13 @@ func (e *EndorsementHandler) Handle(requestContext *RequestContext, clientContex
 
 	requestContext.Response.Responses = transactionProposalResponses
 	if len(transactionProposalResponses) > 0 {
-		requestContext.Response.Payload = transactionProposalResponses[0].ProposalResponse.GetResponse().Payload
+		responsePayload, err := getResultFromProposalResponse(transactionProposalResponses[0].ProposalResponse)
+		if err != nil {
+			requestContext.Error = err
```
Comment (bestbeforetoday) [inline pkg/client/channel/invoke/txnhandler.go:69]: Is just returning the error OK or does it need to be wrapped as an SDK status error?

File: pkg/client/channel/invoke/txnhandler.go [hunk 2]
```diff
@@ -63,7 +64,13 @@ func (e *EndorsementHandler) Handle(requestContext *RequestContext, clientContex
 
 	requestContext.Response.Responses = transactionProposalResponses
 	if len(transactionProposalResponses) > 0 {
-		requestContext.Response.Payload = transactionProposalResponses[0].ProposalResponse.GetResponse().Payload
+		responsePayload, err := getResultFromProposalResponse(transactionProposalResponses[0].ProposalResponse)
```
Comment (bestbeforetoday) [inline pkg/client/channel/invoke/txnhandler.go:67]: Does this need to fallback to `transactionProposalResponses[0].ProposalResponse.GetResponse().Payload` to deal with transaction error responses, or is that handled somewhere else? Passing all the tests so seems this is OK?

**Review by andrew-coleman — APPROVED** [review]

File: pkg/client/channel/invoke/txnhandler.go (hunk 1 above)
Comment (bestbeforetoday) [reply pkg/client/channel/invoke/txnhandler.go:69]: @troyronda Can you confirm the desired behaviour (see comment above)?

File: pkg/client/channel/invoke/txnhandler.go (hunk 2 above)
Comment (bestbeforetoday) [reply pkg/client/channel/invoke/txnhandler.go:67]: @troyronda Can you confirm the desired behaviour (see comment above)?

**Pull request merged**